packages/sdk-v2/src/sdk/microbridge/react.ts

```diff
@@ -2,7 +2,8 @@ import { noop, sortBy } from "lodash";
 import { BridgeSDK } from "@gooddollar/bridge-app/dist/sdk.js";
 import TokenBridgeABI from "@gooddollar/bridge-contracts/artifacts/contracts/bridge/TokenBridge.sol/TokenBridge.json";
 import bridgeContracts from "@gooddollar/bridge-contracts/release/deployment.json";
-import { TokenBridge } from "@gooddollar/bridge-contracts/typechain-types";
+// import { TokenBridge } from "@gooddollar/bridge-contracts/typechain-types/contracts/bridge/TokenBridge.sol/TokenBridge";
```

Comment:
This is an example of a bad refactor.
Replacing a valid type interface and replaced with a placeholder cast to any?

Also, no relation to social share